downstream/modules/eda/con-custom-credential-types.adoc

```diff
@@ -0,0 +1,58 @@
+[id="eda-custom-credential-types"]
+
+= Custom credential types
+
+As a system administrator, you can define a custom credential type works in ways similar to existing credential types in a standard format using a YAML or JSON-like definition. 
```

Comment:
@jself-sudoku Please review this statement, its missing something. Maybe
```
As a system administrator you can define a custom credential type, which works in ways similar to existing credential types.
```

Reply:
Indeed, it is missing the word "that". Thanks for catching it. Corrected.